pvnet_app/sentry.py

```diff
@@ -0,0 +1,24 @@
+""" Sentry tracer function """
+import os
+
+
+def traces_sampler(sampling_context):
+    """
+    Filter tracing for sentry logs.
+
+    Examine provided context data (including parent decision, if any)
+    along with anything in the global namespace to compute the sample rate
+    or sampling decision for this transaction
+    """
+
+    if os.getenv("ENVIRONMENT", "local") == "local":
```

Comment:
Purely as an FYI as I looked at this at some point when adding it, possibly to Quartz Energy – not sure this is actually doing what we think it is / would like it to, but we can revisit another time for sure as it also didn't seem to be doing any harm as it is